spaces/S000060/properties/P000032.md

```diff
@@ -0,0 +1,14 @@
+---
+space: S000060
+property: P000032
+value: false
+---
+
+Take any countable partition $\{A_n:n<\omega\}$ of $\mathbb R\setminus\mathbb Q$ into Euclidean-dense subsets.
+Then $\mathscr U:=\{A_n\cup\mathbb Q: n<\omega\}$ is
+a countable open cover of $X$. Let $\mathscr V$ be an open refinement of $U$.
+Let $U$ be any neighbourhood of $x\in\mathbb R\setminus\mathbb Q$. The closure of $U$ contains an
+interval around $x$, hence for each $n<\omega$ there exists
+$a_n\in A_n\cap \overline{U}$. Since each $a_n$ is contained in a different element of $\mathscr V$,
+$U$ intersects infinitely many elements of the cover
+proving it is not locally finite.
```

Comment:
```suggestion
$a_n\in A_n\cap \overline{U}$.
If $a_n\in V\in\mathscr V$, the open set $V$ intersects $U$.
So, since each $a_n$ is contained in a different element of $\mathscr V$,
$U$ intersects infinitely many elements of $\mathscr V$,
proving $\mathscr V$ is not locally finite.
```
one extra sentence for clarification.  Plus clarify which cover we are talking about at the end.